src/routes/(app)/documents/governing/+page.server.ts

```diff
@@ -5,17 +5,38 @@ import { zod } from "sveltekit-superforms/adapters";
 import { z } from "zod";
 import * as m from "$paraglide/messages";
 
-export const load: PageServerLoad = async ({ locals }) => {
+export const load: PageServerLoad = async ({ locals, url }) => {
   const { prisma } = locals;
+  let year = url.searchParams.get("year") || new Date().getFullYear();
+  year = typeof year === "string" ? parseInt(year) : year;
   const governingDocuments = await prisma.document
-    .findMany({ where: { deletedAt: null } })
-    .then((documents) => ({
-      policies: documents.filter((document) => document.type === "POLICY"),
-      guidelines: documents.filter((document) => document.type === "GUIDELINE"),
-    }));
+    .findMany({
+      where: {
+        deletedAt: null,
+      },
```

Comment:
I don't think this is needed - we have access policies defined in our `schema.zmodel` file that prevents users from reading deleted documents.